test/parallel/test-setproctitle.js

```diff
@@ -21,6 +21,10 @@ assert.notStrictEqual(process.title, title);
 process.title = title;
 assert.strictEqual(process.title, title);
 
+// Test setting the title but do not try to run `ps` on Windows.
```

Comment:
So I naively thought that it should be possible to do the equivalent (i.e. validate the process title change) on Windows via PowerShell. Turns out it's more complicated because on Windows changing the process title changes the title of the console Window which is not necessarily the same process as the node.exe process (e.g. if you open and cmd.exe Window and then launch node.exe inside it, changing the process title in Node.js changes the Window title of the cmd.exe process). 

Anyway I ended up with this hideous command line/PowerShell script, which given a process id will walk up the parent process ids until it finds a process with a non-empty Window title and then print it out:
```PowerShell
powershell "$p = <pid>; $t = Get-Process -pid $p ; while ($t.MainWindowTitle -eq '') { $p = Get-WmiObject win32_process | where {$_.processId -eq $p } | select -ExpandProperty parentProcessId; $t = Get-Process -pid $p } Write-Host $t.MainWindowTitle"
```
Of course `Get-Process` doesn't know about parent process ids, so the script has to use `Get-WmiObject win32_process` (which doesn't know about Window titles).

I make no claims to being a PowerShell expert. Quite happy to submit this as a separate PR.
cc @nodejs/platform-windows